src/main/resources/db/changelog/changes/02-create-users-table.yaml

```diff
@@ -0,0 +1,42 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-table
+      author: Roma Menzheres
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: first_name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: last_name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: email
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+              - column:
+                  name: password
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: role
+                  type: enum('CUSTOMER', 'MANAGER')
+                  constraints:
+                    nullable: false
+
```

Comment:
```suggestion
              - column:
                  name: role
                  type: enum('CUSTOMER', 'MANAGER')
                  constraints:
                    nullable: false
```

Reply:
Weird Line breaking is my vision of readable code. I dont use any code format plugin, so u can show me where it is weird :)